_includes/utilities/responsive-variants.html

```diff
@@ -63,7 +63,7 @@ <h4 class="font-lang-2xs margin-top-2 margin-bottom-05">Output</h4>
     </tbody>
   </table>
 
-  <p class="utility-text margin-top-4">Set whether a utility family outputs with responsive variants by setting the value of  <code>$[utility_family]-settings.responsive</code> in <code>_uswds-theme-utilities.scss</code>.</p>
+  <p class="utility-text margin-top-4">Set whether a utility family outputs with responsive variants by setting the value of <code>$[utility_family]-settings.responsive</code> in your <a href="{{ site.baseurl }}/documentation/settings" class="">USWDS settings configuration</a>.</p>
```

Comment:
This might also benefit from a code example showing nested settings.

Reply:
HA! Made the comment above before reading this. I'll make these changes and lookout for other areas we can improve understanding in a similar way